File: packages/cordova/src/android/cordova/ads/WebViewAd.java
package admob.plus.cordova.ads;

import android.util.Log;

import admob.plus.cordova.ExecuteContext;
import admob.plus.core.Context;

public class WebViewAd extends AdBase {
    private static final String TAG = "AdMobPlus.WebViewAd";

    public WebViewAd(ExecuteContext ctx) {
        super(ctx);
        Log.d(TAG, "WebViewAd");
    }

    @Override
    public boolean isLoaded() {
        return false;
    }

    @Override
    public void load(Context ctx) {
        ctx.resolve();
    }

    @Override
    public void show(Context ctx) {
        ctx.resolve();
    }

    @Override
    public void hide(Context ctx) {
        ctx.resolve();
    }

}

File: packages/cordova/src/android/cordova/ads/WebViewAd.kt
package admob.plus.cordova.ads

import android.util.Log
import admob.plus.cordova.ExecuteContext

class WebViewAd(ctx: ExecuteContext) : AdBase(ctx) {
    init {
        Log.d(TAG, "WebViewAd")
    }

    override val isLoaded: Boolean
        get() = false

    override fun load(ctx: ExecuteContext) {
        ctx.resolve()
    }

    override fun show(ctx: ExecuteContext) {
        ctx.resolve()
    }

    override fun hide(ctx: ExecuteContext) {
        ctx.resolve()
    }

    companion object {
        private const val TAG = "AdMobPlus.WebViewAd"
    }
}
